server_modules/log.js

```diff
@@ -12,7 +12,7 @@ exports.middleware = function(req, res, next) {
 
     req.log = exports.log;
 
-    req.log.debug('Starting {0} {1}'.format(req.method, req.originalUrl));
+    req.log.debug(`Starting ${req.method} ${req.originalUrl}`);
```

Comment:
Slightly unrelated to this PR, but instead of (or in addition to) logging the start of a request, it would be helpful to log the completion of a request because that can include the reponse code.

Reply:
So I looked it up, and it is possible. You just need to add next() calls to all the routing functions. It will be a separate PR though